bskyweb/cmd/bskyweb/rss.go

```diff
@@ -82,7 +82,7 @@ func (srv *Server) WebProfileRSS(c echo.Context) error {
 		}
 	}
 
-	af, err := appbsky.FeedGetAuthorFeed(ctx, srv.xrpcc, did.String(), "", "posts_no_replies", 30)
+	af, err := appbsky.FeedGetAuthorFeed(ctx, srv.xrpcc, did.String(), "", "posts_no_replies", false, 30)
```

Comment:
This was added by Bryan in https://github.com/bluesky-social/social-app/pull/8442/files#diff-4a78d2d0b99a71abd5e7370dd4c984ad576e033485c067db5fa6a24677d76a37, should not be in this PR